app/controllers/shelters_controller.rb

```diff
@@ -32,6 +32,13 @@ def update
 
   def pets
     @shelter = Shelter.find(params[:id])
+    if params[:adoption_status] == "true"
+      @pets = @shelter.pets.adoptable_only
+    elsif params[:adoption_status] == "false"
+      @pets = @shelter.pets.pending_only
+    else
+      @pets = @shelter.pets.order_by_adoption_status
+    end
```

Comment:
I decided to add the `@pets` again because I would rather do this filtering in the controller than in the view.